nodejs/week2/mealSharing/src/bakckend/routes/reservations.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+const router = express.Router();
+const reservations = require("../data/reservations.json");
+
+router.get("/:id", (req, res) => {
+	const reserveId = parseInt(req.params.id);
+	if (!reserveId) {
+		res.send(`Enter an id number`);
```

Comment:
In REST there's a formal way in which to tell a client that the parameters of the request are invalid.

Try to see how to do this with express: Send a 400 bad request error response